atomic_reactor/plugins/pre_pyrpkg_fetch_artefacts.py

```diff
@@ -10,13 +10,41 @@
 
 This plugin should do it.
 """
-import os
+import re
 import subprocess
+from pathlib import Path
+from typing import Iterable, List
 
+from atomic_reactor.dirs import BuildDir
 from atomic_reactor.plugin import PreBuildPlugin
 from atomic_reactor.constants import PLUGIN_DISTGIT_FETCH_KEY
 
 
+MD5_CHECKSUM_RE = r'^[a-f0-9]+ [* ](.+)\n?$'
+BSD_CHECKSUM_RE = r'^[^ ]+ \((.+)\) = [a-f0-9]+\n?$'
```

Comment:
sha512 works just fine with space at the beginning

we should really make correct regex here ..

or in worst case we could just use output of rhpkg sources

OR even better would actually be to run rhpkg sources in NEWLY created empty directory, and just use file list from there !

Reply:
Do we use real, actual, [BSD checksum](https://en.wikipedia.org/wiki/BSD_checksum)[1], or is this just an odd name?

[1] If so, why?

Reply:
@rcerven 

This regex actually follows the one used in rpkg. The lines of sources are not generated by sha512sum command, instead they are written by the rpkg code which does not put a space at the beginning.